src/components/SortBy/SortBy.jsx

```diff
@@ -0,0 +1,18 @@
+import './SortBy.css'
+
+export const SortBy = ({sortOptions, setSortItemsBy, isAscOrder, setIsAscOrder}) => {
+    return (
+        <menu className="sort-by-menu">
+            <ul>
```

Comment:
I would be nice if the user was given an indeication of what these buttons do, maybe just a label that said sort by or something?